pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -82,7 +82,7 @@ func (pl *VolumeZone) Filter(ctx context.Context, _ *framework.CycleState, pod *
 	}
 	nodeConstraints := make(map[string]string)
 	for k, v := range node.ObjectMeta.Labels {
-		if k != v1.LabelZoneFailureDomain && k != v1.LabelZoneRegion {
```

Comment:
we need to support both old and stable labels, so you should keep the checks for LabelZoneFailureDomain and LabelZoneRegion

Reply:
But I see there is some reconciliation code in https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/cloud/node_controller.go#L203-L211

Maybe the new labels will always be generated?

Reply:
Oh, seems the persistent volume won't handle this.

Reply:
Thanks for pointing to that code, I didn't know it existed. Also, from the reconciliation code it seems that the beta labels are added if not exist, so not checking the stable labels is not really a bug, yet.

Lets check for both just to be consistent with other places in the code, this is easy to remove once those constants are removed.

Reply:
Actually it ensures the existence of GA labels instead of the beta labels. So I think we need to cherry-pick this to 1.17 and 1.18

Reply:
nodes still set both the beta and GA labels, I would not expect this to be picked into old releases